File: programmers/lv0/코딩기초트레이닝/연산/더크게합치기/Solution.cpp
#include <string>

using namespace std;

int solution(int a, int b) {
    int x = stoi(to_string(a) + to_string(b));
    int y = stoi(to_string(b) + to_string(a));
    if (x == y) {
        return x;
    }
    if (x > y) return x;
    return y;
}

File: programmers/lv0/코딩기초트레이닝/연산/더크게합치기/Solution.kt
package lv0.코딩기초트레이닝.연산.더크게합치기

import kotlin.math.max

fun solution(a: Int, b: Int): Int {
//    val x = (a.toString() + b.toString()).toInt()
//    val y = (b.toString() + a.toString()).toInt()
//    return Math.max(x, y)
    return max("$a$b".toInt(), "$b$a".toInt())
}
